src/core/server/http/http_service.ts

```diff
@@ -198,15 +198,13 @@ export class HttpService
       path: '/{p*}',
       method: '*',
       handler: (req, responseToolkit) => {
-        this.log.debug(
-          `OpenSearch Dashboards server is not ready yet ${req.method}:${req.url.href}.`
-        );
+        this.log.debug(`Wazuh Dashboard server is not ready yet ${req.method}:${req.url.href}.`);
 
         // If server is not ready yet, because plugins or core can perform
         // long running tasks (build assets, saved objects migrations etc.)
         // we should let client know that and ask to retry after 30 seconds.
         return responseToolkit
-          .response('OpenSearch Dashboards server is not ready yet')
+          .response('Wazuh Dashboard server is not ready yet')
```

Comment:
nitpick: Dashboard should be in lowercase.

Reply:
WIP. To be discussed.